dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -15,44 +15,22 @@
 from dask_jobqueue import (
     JobQueueCluster,
     PBSCluster,
-    MoabCluster,
-    SLURMCluster,
-    SGECluster,
-    LSFCluster,
-    OARCluster,
     HTCondorCluster,
 )
 from dask_jobqueue.core import Job
 from dask_jobqueue.local import LocalCluster
 
 from dask_jobqueue.sge import SGEJob
 
-all_clusters = [
-    PBSCluster,
-    MoabCluster,
-    SLURMCluster,
-    SGECluster,
-    LSFCluster,
-    OARCluster,
-    HTCondorCluster,
-]
-
-
-def create_cluster_func(cluster_cls, **kwargs):
-    if cluster_cls is HTCondorCluster:
-        # HTCondorCluster has a mandatory disk argument
-        kwargs.setdefault("disk", "1GB")
-    return cluster_cls(**kwargs)
-
 
 def test_errors():
     match = re.compile("Job type.*job_cls", flags=re.DOTALL)
     with pytest.raises(ValueError, match=match):
         JobQueueCluster(cores=4)
 
 
-def test_command_template():
-    with PBSCluster(cores=2, memory="4GB") as cluster:
+def test_command_template(Cluster):
+    with Cluster(cores=2, memory="4GB") as cluster:
```

Comment:
I tried to make that test, that run across all clusters are actually tested across all clusters (or in rare cases only `LocalCluster`)